### PR TITLE
qtpbfimageplugin: init at 1.4

### PR DESCRIPTION
**Pull request by sikmir**
###### Motivation for this change

Optional dependency for [gpxsee](https://github.com/NixOS/nixpkgs/tree/master/pkgs/applications/misc/gpxsee) that enables support for PBF vector maps.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by sikmir** [comment]
@dywedir FYI, gpxsee depends on it.

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -4671,6 +4671,11 @@
     github = "sigma";
     name = "Yann Hodique";
   };
+  sikmir = {
+    email = "sikmir@gmail.com";
+    github = "sikmir";
+    name = "Nikolay Korotkiy";
+  };
```
Comment (jonringer) [inline maintainers/maintainer-list.nix]: IIRC, adding yourself as a maintainer is preferred to be a separate commit.

Something like, `maintainers: add sikmir`. Then apply the qtpbfimageplugin commit.

Other than that, `nix-review pr 63506` passed, and LGTM otherwise

Comment (sikmir) [reply maintainers/maintainer-list.nix]: Thanks, fixed.

**Comment by sikmir** [comment]
@jonringer Is there any more todos here? Otherwise, can we land this PR?

**Comment by worldofpeace** [comment]
@GrahamcOfBorg build qtpbfimageplugin

File: pkgs/development/libraries/qtpbfimageplugin/default.nix [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    description = "Qt image plugin for displaying Mapbox vector tiles";
+    longDescription = ''
+      QtPBFImagePlugin is a Qt image plugin that enables applications capable of
+      displaying raster MBTiles maps or raster XYZ online maps to also display PBF
+      vector tiles without (almost) any application modifications.
+    '';
+    homepage = https://github.com/tumic0/QtPBFImagePlugin;
+    license = licenses.lgpl3;
+    maintainers = [ maintainers.sikmir ];
+    platforms = platforms.linux;
```
Comment (worldofpeace) [inline pkgs/development/libraries/qtpbfimageplugin/default.nix]:
```suggestion
    platforms = platforms.unix;
```

Darwin is supported https://github.com/tumic0/QtPBFImagePlugin/blob/master/.travis.yml#L6

Comment (sikmir) [reply pkgs/development/libraries/qtpbfimageplugin/default.nix]: Fixed.

**Comment by worldofpeace** [comment]
@GrahamcOfBorg build qtpbfimageplugin

**Comment by worldofpeace** [comment]
@sikmir I've fixed the darwin build for you.
Though I have a glimmering that it's probably an issue to fix upstream.

We can merge this now :confetti_ball:
